### PR TITLE
Updated more meta file and added some tracks

### PR DESCRIPTION
**Pull request by SonicRPika**
Added the following:

Sonic the Hedgehog 2 (8 bit) - Green Hills Zone
Sonic Chaos - Mecha Green Hill Zone
Sonic CD (JP) - You Can Do Anything
Pokémon Scarlet/Violet - Battle Penny

Also updated some of the Sonic tracks I have made to have groups. Haven't done them all yet though

### CODE REVIEW

**Review by TheSoundDefense — COMMENTED** [review]
This PR adds a second "Endless Possibility" song with the exact same title as the first. This breaks the randomizer. Please double check this.

**Comment by SonicRPika** [comment]
Hope this fixes this

File: data/Music/Pokemon Series/Scarlet and Violet/Battle! vs Penny.meta [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Pokémon Scarlet/Violet - Battle! vs Penny
```
Comment (TheSoundDefense) [inline data/Music/Pokemon Series/Scarlet and Violet/Battle! vs Penny.meta:1]: This song is close to the line of being too loud. You might want to bring the volume down just a bit.

File: data/Music/Sonic the Hedgehog Series/Sonic 2 (8 bit)/Green Hills Zone.meta [hunk 2]
```diff
@@ -0,0 +1,4 @@
+Sonic the Hedgehog 2 (8 bit) - Green Hills Zone
+0x03
+bgm
+HyruleField,Fields
```
Comment (TheSoundDefense) [inline data/Music/Sonic the Hedgehog Series/Sonic 2 (8 bit)/Green Hills Zone.meta:4]: The location line should have at least one group from each level of specificity (exact, high, mid and low). We can merge the PR without this, but then someone else will come in and add the missing groups for you, just so you know.

File: data/Music/Sonic the Hedgehog Series/Sonic CD/You Can Do Anything.meta [hunk 3]
```diff
@@ -0,0 +1,4 @@
+Sonic CD (JP) - You Can Do Anything
+0x21
+bgm
+TitleTheme
```
Comment (TheSoundDefense) [inline data/Music/Sonic the Hedgehog Series/Sonic CD/You Can Do Anything.meta:4]: The location line should have at least one group from each level of specificity (exact, high, mid and low).

File: data/Music/Sonic the Hedgehog Series/Sonic CD/You Can Do Anything.meta [hunk 4]
```diff
@@ -0,0 +1,4 @@
+Sonic CD (JP) - You Can Do Anything
```
Comment (TheSoundDefense) [inline data/Music/Sonic the Hedgehog Series/Sonic CD/You Can Do Anything.meta:1]: A couple of notes:
- The Malon voice is kind of buried in the mix, especially near the end. You might want to bring this out more.
- Having the entire song come to an end and then start again feels strange. I would recommend finding another loop point so the music is not constantly interrupted.

File: data/Music/Sonic the Hedgehog Series/Sonic CD/jpbadstardustspeedway.meta [hunk 5]
```diff
@@ -1,3 +1,4 @@
 Sonic CD (JP) - Stardust Speedway Bad Future
 0x03
-bgm
\ No newline at end of file
+bgm
+BigFight,DampeRace
```
Comment (TheSoundDefense) [inline data/Music/Sonic the Hedgehog Series/Sonic CD/jpbadstardustspeedway.meta:4]: The location line should have at least one group from each level of specificity (exact, high, mid and low).

File: data/Music/Sonic the Hedgehog Series/Sonic Chaos/Mecha Green Hill Zone.meta [hunk 6]
```diff
@@ -0,0 +1,4 @@
+Sonic Chaos - Mecha Green Hill Zone
+0x03
+bgm
+HyruleField,Fields,WindmillHut
```
Comment (TheSoundDefense) [inline data/Music/Sonic the Hedgehog Series/Sonic Chaos/Mecha Green Hill Zone.meta:4]: The location line should have at least one group from each level of specificity (exact, high, mid and low).

File: data/Music/Sonic the Hedgehog Series/Sonic Rush/VelaNova.meta [hunk 7]
```diff
@@ -1,3 +1,4 @@
 Sonic Rush - Vela Nova
 0x03
-bgm
\ No newline at end of file
+bgm
+GanonBattle,GanondorfBattle
```
Comment (TheSoundDefense) [inline data/Music/Sonic the Hedgehog Series/Sonic Rush/VelaNova.meta:4]: The location line should have at least one group from each level of specificity (exact, high, mid and low).

File: data/Music/Sonic the Hedgehog Series/Sonic the Hedgehog 1/Fanfare - 1Up.meta [hunk 8]
```diff
@@ -1,3 +1,4 @@
 Sonic the Hedgehog - 1-Up
 0x16
-Fanfare
\ No newline at end of file
+Fanfare
+HeartContainerGet,HeartPieceGet,EventFanfare
```
Comment (TheSoundDefense) [inline data/Music/Sonic the Hedgehog Series/Sonic the Hedgehog 1/Fanfare - 1Up.meta:4]: The location line should have at least one group from each level of specificity (exact, high, mid and low).
```suggestion
HeartContainerGet,HeartPieceGet,ItemFanfare,EventFanfare
```

File: data/Music/Sonic the Hedgehog Series/Sonic the Hedgehog 1/MarbleZone.meta [hunk 9]
```diff
@@ -1,3 +1,4 @@
 Sonic the Hedgehog - Marble Zone
 0x03
-bgm
\ No newline at end of file
+bgm
+DodongosCavern,FireTemple,Dungeon,AncientDungeon
```
Comment (TheSoundDefense) [inline data/Music/Sonic the Hedgehog Series/Sonic the Hedgehog 1/MarbleZone.meta:4]: The location line should have at least one group from each level of specificity (exact, high, mid and low).

File: data/Music/Sonic the Hedgehog Series/Sonic the Hedgehog 1/final.meta [hunk 10]
```diff
@@ -1,3 +1,2 @@
-Sonic the Hedgehog - Final Zone
-0x03
-bgm
```
Comment (TheSoundDefense) [inline data/Music/Sonic the Hedgehog Series/Sonic the Hedgehog 1/final.meta:3]: Were the changes to this file intended?

File: data/Music/Sonic the Hedgehog Series/Sonic the Hedgehog 3/IceCap1.meta [hunk 11]
```diff
@@ -1,3 +1,4 @@
 Sonic the Hedgehog 3 & Knuckles - Ice Cap Act 1
 0x03
-bgm
\ No newline at end of file
+bgm
+IceCavern,ZorasDomain
```
Comment (TheSoundDefense) [inline data/Music/Sonic the Hedgehog Series/Sonic the Hedgehog 3/IceCap1.meta:4]: The location line should have at least one group from each level of specificity (exact, high, mid and low).

File: data/Music/Sonic the Hedgehog Series/Sonic the Hedgehog 3/IceCap2.meta [hunk 12]
```diff
@@ -1,3 +1,4 @@
 Sonic the Hedgehog 3 & Knuckles - Ice Cap Act 2
 0x03
 bgm
+IceCavern,ZorasDomain
```
Comment (TheSoundDefense) [inline data/Music/Sonic the Hedgehog Series/Sonic the Hedgehog 3/IceCap2.meta:4]: The location line should have at least one group from each level of specificity (exact, high, mid and low).

**Review by TheSoundDefense — COMMENTED** [review]
I added some comments that you don't have to address to merge this PR, but I'd like you to take a look at them.

File: data/Music/Sonic the Hedgehog Series/Sonic the Hedgehog 1/final.meta (hunk 10 above)
Comment (SonicRPika) [reply data/Music/Sonic the Hedgehog Series/Sonic the Hedgehog 1/final.meta:3]: I either forgot to update it, or didn't have a good location at the time and forgot to remove it. I have fixed this

**Comment by SonicRPika** [comment]
Addressing most of the comments here:

If I haven't given a location of each specificity, it's because I got confused when looking at other meta files. I am happy for those with some added already to get more added to it (most likely whichever one groups the most added).
I have fixed the sound concern of Battle Penny by reducing the total volume by 8. Hopefully this will help
Fixed my error with one of the meta files as I must have uploaded on accident.

With the You Can Do Anything item, it's technically a song/intro so there isn't really a good loop point for it (same for tracks like Team Fortress 2 - Right Behind You), however I have set all non-vocal parts of the song to be lower with Malon being at max volume

File: data/Music/Sonic the Hedgehog Series/Sonic 2 (8 bit)/Green Hills Zone.meta (hunk 2 above)
Comment (TheSoundDefense) [inline data/Music/Sonic the Hedgehog Series/Sonic 2 (8 bit)/Green Hills Zone.meta:4]:
```suggestion
HyruleField,Fields,Outdoors,Overworld
```

**Comment by TheSoundDefense** [comment]
I'll fix up the location tags for your songs before merging this.

For the future, I wrote a web tool that should take a lot of the confusion out of location tagging. You can try it out [here](https://thesounddefense.github.io/musicgroups/).

File: data/Music/Sonic the Hedgehog Series/Sonic CD/jpbadstardustspeedway.meta (hunk 5 above)
Comment (TheSoundDefense) [inline data/Music/Sonic the Hedgehog Series/Sonic CD/jpbadstardustspeedway.meta:4]:
```suggestion
BossBattle,FireBoss,DodongosCavern,BossFight,BigFight,Fight
```

File: data/Music/Sonic the Hedgehog Series/Sonic Chaos/Mecha Green Hill Zone.meta (hunk 6 above)
Comment (TheSoundDefense) [inline data/Music/Sonic the Hedgehog Series/Sonic Chaos/Mecha Green Hill Zone.meta:4]:
```suggestion
HyruleField,LostWoods,GerudoValley,WindmillHut,Fields,Outdoors,Overworld
```

File: data/Music/Sonic the Hedgehog Series/Sonic Rush/VelaNova.meta (hunk 7 above)
Comment (TheSoundDefense) [inline data/Music/Sonic the Hedgehog Series/Sonic Rush/VelaNova.meta:4]:
```suggestion
GanondorfBattle,GanonBattle,FinalFight,BigFight,Fight
```

File: data/Music/Sonic the Hedgehog Series/Sonic the Hedgehog 1/MarbleZone.meta (hunk 9 above)
Comment (TheSoundDefense) [inline data/Music/Sonic the Hedgehog Series/Sonic the Hedgehog 1/MarbleZone.meta:4]:
```suggestion
DodongosCavern,FireTemple,ChildDungeon,AncientDungeon,AdultDungeon,Dungeon
```

File: data/Music/Sonic the Hedgehog Series/Sonic the Hedgehog 1/final.meta [hunk 13]
```diff
@@ -1,3 +1,4 @@
-Sonic the Hedgehog - Final Zone
+Sonic the Hedgehog 1 - Final Zone
```
Comment (TheSoundDefense) [inline data/Music/Sonic the Hedgehog Series/Sonic the Hedgehog 1/final.meta:1]:
```suggestion
Sonic the Hedgehog - Final Zone

```

File: data/Music/Sonic the Hedgehog Series/Sonic the Hedgehog 1/final.meta [hunk 14]
```diff
@@ -1,3 +1,5 @@
 Sonic the Hedgehog - Final Zone
+
 0x03
-bgm
\ No newline at end of file
+bgm
+EndFight,BossBattle,FireBoss,GanondorfBattle,GanonBattle
```
Comment (TheSoundDefense) [inline data/Music/Sonic the Hedgehog Series/Sonic the Hedgehog 1/final.meta:5]:
```suggestion
GanondorfBattle,GanonBattle,FinalFight,BigFight,Fight
```

File: data/Music/Sonic the Hedgehog Series/Sonic the Hedgehog 3/IceCap1.meta (hunk 11 above)
Comment (TheSoundDefense) [inline data/Music/Sonic the Hedgehog Series/Sonic the Hedgehog 3/IceCap1.meta:4]:
```suggestion
ZorasDomain,IceCavern,MysticalDungeon,AdultDungeon,Dungeon
```

File: data/Music/Sonic the Hedgehog Series/Sonic the Hedgehog 3/IceCap2.meta (hunk 12 above)
Comment (TheSoundDefense) [inline data/Music/Sonic the Hedgehog Series/Sonic the Hedgehog 3/IceCap2.meta:4]:
```suggestion
ZorasDomain,IceCavern,MysticalDungeon,AdultDungeon,Dungeon
```

File: data/Music/Sonic the Hedgehog Series/Sonic the Hedgehog 3/Sando2.meta [hunk 15]
```diff
@@ -1,3 +1,4 @@
 Sonic the Hedgehog 3 & Knuckles - Sandopolis Zone Act 2
 0x03
-bgm
\ No newline at end of file
+bgm
+SpiritTemple,CastleUnderground,ShadowTemple,ForestTemple,Dungeon,AdultDungeon,MysticalDungeon
```
Comment (TheSoundDefense) [inline data/Music/Sonic the Hedgehog Series/Sonic the Hedgehog 3/Sando2.meta:4]:
```suggestion
ForestTemple,SpiritTemple,ShadowTemple,CastleUnderground,AncientDungeon,MysticalDungeon,SpookyDungeon,AdultDungeon,Dungeon
```

**Pull request merged**